tr7zw/YAPFA
# Nuke more streams

How to test:
1. Try throwing a fishing rod and see if everything's fine with that
2. Try teleporting entities (including yourself) at locations
3. Try interacting with blocks and moving in general
